File: Semana 1/Clase 2/EjemploC1/Circulo.cs
public class Circulo : FiguraGeometrica{
	private double radio;

	public Circulo(double radio): base("Circulo"){
		this.radio = radio;
	}

	public double gatRadio(){
		return radio;
	}

	public override double calcularArea(){
		return Math.PI * radio * radio;
	}

	public override double calcularPerimetro(){
		return 2 * Math.PI * radio;
	}

	

}

File: Semana 4/PaquetesJava/src/com/pucp/Circulo.java
package com.pucp;

public class Circulo extends FiguraGeometrica {
	private double radio;

	public Circulo(double radio){
		super("Circulo");
		this.radio = radio;
	}

	@Override
	public double calcularArea(){
		return Math.PI * radio * radio;
	}

	@Override
	public double calcularPerimetro(){
		return 2 * Math.PI * radio;
	}

	@Override
	public String toString(){
		return getNombre() + ": ("+radio+")";
	}

}